test/integration/guest_user_test.rb

```diff
@@ -144,14 +144,16 @@ class GuestUserTest < ActionDispatch::IntegrationTest
     end
   end
 
-  test "an unauthorized user can add projects to pending_loans but cannot
-  checkout page logging in" do
-    tenant = create(:tenant)
-    tenant.projects << create(:project, price: 8900)
-
+  test "after an unauthorized user adds items and clicks checkout and logs in,
```

Comment:
Since the user story I'm on relates to an unauthorized user, I think leaving it here but rearranging the order of what happens in the test would be better.